Poker.BE/Poker.BE.Domain/Game/Round.cs

```diff
@@ -25,69 +26,256 @@ public enum Move
         #region fields
         private ICollection<Player> activeUnfoldedPlayers;
         private Turn currentTurn;
-        private Player dealer;
+        //private Player dealer;        Is dealer needed??
```

Comment:
haha I'll tell you what I meant:
the round doesn't know if it's the pre-flop round or a regular round, so if it just get the dealer it doesn't know where to start (at pre-flop you start at the player who sits after the big blind, not like other rounds).
therefore, it's better that the round will get the starting player immediately (into the currentPlayer field), it doesn't care about who's the dealer if it already know where to start.